openrlhf/datasets/sft_dataset.py

```diff
@@ -148,23 +150,29 @@ def collate_fn(self, item_list):
 
         input_ids = zero_pad_sequences(input_ids, "right", self.tokenizer.pad_token_id)
         attention_masks = zero_pad_sequences(attention_masks, "right")
-        return prompt_ids_lens, input_ids, attention_masks, infos
+        return prompt_ids_lens, input_ids, attention_masks, None, infos
+
 
     def packing_collate_fn(self, item_list):
-        packed_input_ids = []
-        packed_attention_masks = []
-        prompt_ids_lens = []
+        packed_input_ids, packed_attention_masks = [], []
+        prompt_ids_lens, packed_seq_lens = [], []
```

Comment:
please keep the origin code and add a newline for `packed_seq_lens`
```suggestion
        packed_seq_lens = []
```